rebasebot/bot.py

```diff
@@ -140,9 +173,7 @@ def _resolve_rebase_conflicts(gitwd):
         if not _resolve_conflict(gitwd):
             return False
 
-        logging.info("Rebase conflict has been resolved. Continue rebasing.")
-
-        gitwd.git.rebase("--continue")
```

Comment:
is this continue no longer needed? (i didn't see anywhere else that does it, and i wasn't quite sure of the git state at this point)

Reply:
Because we are cherry-picking commit by commit now, the resolve conflicts code finishes the cherry-pick by resolving the issues and committing them.

In my testing, the only scenario I saw that caused issues here was empty commits, which the code now correctly drops. Will double check what happens when there are code conflicts

Reply:
ack, thanks